src/components/Error.vue

```diff
@@ -23,8 +23,8 @@
           <span id="error-message" aria-live="polite" role="alert">{{ message || 'Uh oh, there was a problem.' }}</span>
         </div>
         <div>
-          Problem? Question?
-          Email us at <a id="help-mailto" :href="`mailto:${emailAddressSupport}`" target="_blank">{{ emailAddressSupport }}</a>.
+          Problem? Question? Please review our <a id="help-article-kb" :href="`${kbArticleUrl}`" target="_blank">support documentation</a>
```

Comment:
All looks good @mlincoln .  Fyi, `:href="`${kbArticleUrl}`"` can be simplified to `:href="kbArticleUrl"`.  Amendment not required.